exporter/metrics.go

```diff
@@ -11,27 +11,27 @@ func AddMetrics() map[string]*prometheus.Desc {
 	APIMetrics["Stars"] = prometheus.NewDesc(
 		prometheus.BuildFQName("github", "repo", "stars"),
 		"Total number of Stars for given repository",
-		[]string{"repo", "user", "private"}, nil,
+		[]string{"repo", "user", "private", "fork", "arhived", "license", "language"}, nil,
 	)
 	APIMetrics["OpenIssues"] = prometheus.NewDesc(
 		prometheus.BuildFQName("github", "repo", "open_issues"),
 		"Total number of open issues for given repository",
-		[]string{"repo", "user", "private"}, nil,
+		[]string{"repo", "user", "private", "fork", "arhived", "license", "language"}, nil,
 	)
 	APIMetrics["Watchers"] = prometheus.NewDesc(
 		prometheus.BuildFQName("github", "repo", "watchers"),
 		"Total number of watchers/subscribers for given repository",
-		[]string{"repo", "user", "private"}, nil,
+		[]string{"repo", "user", "private", "fork", "arhived", "license", "language"}, nil,
```

Comment:
Shouldn't `arhived` in all these rather be `archived`? (minor typo)

Reply:
good catch, fixed, thanks!